doc/source/theory/multiphase/cfd_dem/dem.rst

```diff
@@ -6,23 +6,23 @@ In this guide, we summarize the theory behind DEM. For further details, we refer
 
 
 .. math::
```

Comment:
The "subscript" or "superscript" should be in italic if it's a variable name (e.g t for time) and in \mathrm when it is an abbreviation. Just double check that this is what youd id here and that you did not put everything in mathrm.